prov/rxm/src/rxm_ep.c

```diff
@@ -37,54 +37,68 @@
 
 #include "rxm.h"
 
-#define RXM_EP_RECV_SANITIZE_SRC_ADDR(rxm_ep, src_addr)	\
-	((rxm_ep->rxm_info->caps & FI_DIRECTED_RECV) ?	\
-	 src_addr : FI_ADDR_UNSPEC)
-
 const size_t rxm_pkt_size = sizeof(struct rxm_pkt);
 
+static int rxm_match_noop(struct dlist_entry *item, const void *arg)
+{
+	OFI_UNUSED(item);
+	OFI_UNUSED(arg);
+	return 1;
+}
+
 static int rxm_match_recv_entry(struct dlist_entry *item, const void *arg)
 {
 	struct rxm_recv_match_attr *attr = (struct rxm_recv_match_attr *) arg;
-	struct rxm_recv_entry *recv_entry;
-
-	recv_entry = container_of(item, struct rxm_recv_entry, entry);
+	struct rxm_recv_entry *recv_entry =
+		container_of(item, struct rxm_recv_entry, entry);
 	return rxm_match_addr(recv_entry->addr, attr->addr);
 }
 
-static int rxm_match_recv_entry_tagged(struct dlist_entry *item, const void *arg)
+static int rxm_match_tag_only_recv_entry_tagged(struct dlist_entry *item, const void *arg)
```

Comment:
nit: how about renaming this function to `rxm_match_recv_entry_tag` and the below one to `rxm_match_recv_entry_tag_addr`?

Reply:
Done